openassessment/xblock/static/js/src/studio/oa_edit_listeners.js

```diff
@@ -277,6 +277,15 @@ export class StudentTrainingListener {
       .text(data.label);
   }
 
+  /**
+   * Actions to take when a rubric is cloned/replaced. Currently this is:
+   *  - Clear training examples
+   */
+  rubricReplaced() {
+    const trainingExamples = $('#openassessment_training_example_list .openassessment_training_example');
```

Comment:
Make sure this is just searching within the target rubric.

Reply:
To ensure it is select within, you should pass `this.elements` as the second argument for this selector.